Resources/Prototypes/_RMC14/Actions/Xeno/xeno_offense_actions.yml

```diff
@@ -513,3 +513,17 @@
     - ActionXenoSprayAcidBoiler
     cooldown: 30
     onPerform: false
+
+- type: entity
+  id: ActionXenoTremor
+  parent: ActionXenoBase
+  name: Tremor (100)
+  description: Slam the ground with a 7x7 AOE, stuning all enemies in range for a short period of time
```

Comment:
Should be "stunning" not stuning, also needs period at the end, otherwise is good.